docs_user/modules/proc_adopting-the-placement-service.adoc

```diff
@@ -2,21 +2,20 @@
 
 = Adopting the Placement service
 
+To adopt the Placement service, you patch an existing `OpenStackControlPlane` custom resource (CR) that has the Placement service disabled. The patch starts the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) environment.
+
 .Prerequisites
 
-* Previous Adoption steps completed. Notably,
- ** the xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
-must already be imported into the control plane MariaDB.
- ** the xref:adopting-the-identity-service_adopt-control-plane[Adopting the Identity service] needs to be imported.
- ** the Memcached operator needs to be deployed (nothing to import for it from
-the source environment).
+* You import your databases to MariaDB instances on the control plane. For more information, see xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances].
+* You adopt the {identity_service_first_ref} to the {rhos_long} the xref:adopting-the-identity-service_adopt-control-plane[Adopting the Identity service].
```

Comment:
Can these prerequisites be removed? Customers should be following the guide in sequence, so they would have done these steps already. But if there’s a prerequisite that you think we should emphasize, please let me know.

Reply:
i think they can be removed however the guide was not originally intended to have a strictly defined order 

note that some service have depencies one other and each service section was originally intended to be completed independently.


if you want the doc to be followed in order then you need to do the adtoping in depency order not simply alpahpetically for example

placement depends on mariadb, and keystone

so keystone must be adopted before placement. nova depend on placement so placement should be adopted before nova is.

placement does not depend on memcache so 
```
You must deploy the Memcached operator. For more information, see xref:deploying-backend-services_migrating-databases[Deploying back-end services].
```
is incorrect

Reply:
keeping preresuisites in line, helps to avoid breakages caused by moving the docs pages around. But I have no strong opinion here. Just to note that that should be a global decision to be made

Reply:
@bogdando Can I remove the prerequisite to deploy the Memcached operator? It was commented that it is incorrect.